components/consumers/consumer.go

```diff
@@ -5,47 +5,49 @@ package consumers
 import (
 	"flag"
 	"fmt"
+	"log/slog"
+	"os"
 
-	v1 "github.com/ocurity/dracon/api/proto/v1"
+	draconapiv1 "github.com/ocurity/dracon/api/proto/v1"
+	"github.com/ocurity/dracon/components"
 
+	draconLogger "github.com/ocurity/dracon/pkg/log"
 	"github.com/ocurity/dracon/pkg/putil"
 )
 
-const (
-	// EnvDraconStartTime Start Time of Dracon Scan in RFC3339.
-	EnvDraconStartTime = "DRACON_SCAN_TIME"
-	// EnvDraconScanID the ID of the dracon scan.
-	EnvDraconScanID = "DRACON_SCAN_ID"
-	// EnvDraconScanTags the tags of the dracon scan.
-	EnvDraconScanTags = "DRACON_SCAN_TAGS"
-)
-
 var (
 	inResults string
 	// Raw represents if the non-enriched results should be used.
 	Raw bool
+	// Debug flag initializes the logger with a debug level
+	Debug bool
```

Comment:
why is Raw and Debug public over here? who needs access to the values of these variables? `Raw` also doesn't seem to be used at all